data.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const mysql = require('mysql');
+
+
+const con = mysql.createConnection({
+
+    host: 'localhost',
+    user: 'root',
+    password: 'kvnmaria@12',
+    database: 'leave_application'
+
+})
+
+function databaseConnection() {
+    return new Promise((resolve, reject) => {
+
+        con.connect((err) => {
+            if (!err) {
+                resolve(true)
+            } else {
+                reject(err)
+            }
+        })
+
+    })
+}
+
+const init = async () => {
+
+    const app = express();
+
+    app.use(express.json());
+    app.use(express.urlencoded({ extended: true }));
+
+    await databaseConnection();
+
+
+    app.post('/employee', (req, res) => {
+
+        const id = req.body.id;
+        const name = req.body.name;
+        const mail = req.body.mail;
+        const role = req.body.role;
+        const mobile = req.body.mobile;
+
+
+        if (!id || !name || !mail || !role || !mobile) {
```

Comment:
Please be specific in which field is missing. For example if only email is missing you should email missing